agents/silicon_valley/main.py

```diff
@@ -0,0 +1,28 @@
+import fixieai
+
+BASE_PROMPT = """I am an agent that answers questions about the TV show Silicon Valley."""
+
+FEW_SHOTS = """
+Q: Who was Gilfoyle played by?
```

Comment:
@hessamb it would be great to have some knob that allowed PromptRunner to just work as a Q/A agent with no few-shots